docs/assistant/security-assistant.asciidoc

```diff
@@ -177,4 +177,17 @@ To enable the knowledge base:
 . Turn on the *Knowledge Base* option.
 . Click *Save*. The knowledge base is now active.
 +
-When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
\ No newline at end of file
+When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
+
+[discrete]
+[[ai-assistant-queries]]
+### How to get the most from your queries
+
+The goal of the AI Assistant is to help users take full advantage of the Elastic Security platform to improve their cyber operations. Its ability to assist you depends on the specificity and detail of your questions. The more context and detail you provide, the more tailored and useful its responses will be. 
+
+To maximize its usefulness, consider using more detailed or elaborate prompts. For instance, after asking for an {esql} query example, you could ask a follow-up question like, “Could you give me some other examples and provide comments explaining the query you just gave?”. You can also ask for clarification or further exposition. 
```

Comment:
Nit: "Elaborate" can have a slightly negative connotation; I think just "more detailed" is enough to make this point.

Also the example prompts here don't really include _more details_ in the prompts themselves; they're just asking for additional information. They're also asking for two different kinds of info (more examples, and explain the original example), so it'd help to mention each one separately.

```suggestion
To maximize its usefulness, consider using more detailed prompts or asking for additional information. For instance, after asking for an {esql} query example, you could ask a follow-up question like, “Could you give me some other examples?” You can also ask for clarification or further exposition, such as "Could you provide comments explaining the query you just gave?"
```